kafka/adminapi_test.go

```diff
@@ -30,6 +30,11 @@ func TestAdminAPIWithDefaultValue(t *testing.T) {
 		t.Skipf("Missing testconf.json")
 	}
 
+	stateErr := Error{
```

Comment:
Why don't we use `newErrorFromString` method here to create the error? Would be good since any changes in that method would otherwise require changing this value, too.